data/json/items/armor.json

```diff
@@ -2974,7 +2974,7 @@
         "environmental_protection" : 20,
         "encumbrance" : 37,
         "bashing" : -5,
-        "flags" : ["VARSIZE", "WATERPROOF", "RAINPROOF", "RAD_PROOF","ELECTRIC_IMMUNE", "OUTER"],
+        "flags" : ["VARSIZE", "WATERPROOF", "RAINPROOF", "RAD_PROOF", "ELECTRIC_IMMUNE", "GAS_PROOF", "OUTER"],
```

Comment:
GAS_PROOF would be better as an environmental protection formula (so that you could partially resist it), but it's OK as a tag for now.
When (if?) you add more chemical types later it may be worth considering adding a gas resistance check as a separate function, but probably not as a part of this PR.